.github/workflows/update-kernels.yml

```diff
@@ -21,13 +42,37 @@ jobs:
         with:
           ref: kernels
 
+      - name: Setup jq
+        run: apt-get update && apt-get install jq -y
+
       - name: Run crawler for x86_64
         run: |
-          kernel-crawler crawl --distro=* > x86_64/list.json
+          mkdir $RUNNER_TEMP/x86_64
+          kernel-crawler crawl --distro=${{ inputs.distro }} > $RUNNER_TEMP/x86_64/list.json
```

Comment:
When we run on `schedule` trigger, we haven't got any `inputs.distro`.
Same below for arm64.